src/tools/sss_cache.c

```diff
@@ -147,6 +147,21 @@ int main(int argc, const char *argv[])
     bool skipped = true;
     struct sss_domain_info *dinfo;
 
+    /* In offline mode, there's not going to be a sssd instance
```

Comment:
Sumit mentioned that the offline mode here is confusing because it's used in sssd as well in a differnet context, so maybe we should clarify it's systemd offline mode